helpers/gradio_demo/app.py

```diff
@@ -37,14 +37,15 @@
 
 
 def gen_tts(text, description):
-    inputs = tokenizer(description, return_tensors="pt").to(device)
-    prompt = tokenizer(text, return_tensors="pt").to(device)
+    with torch.no_grad():
+        inputs = tokenizer(description, return_tensors="pt").to(device)
+        prompt = tokenizer(text, return_tensors="pt").to(device)
 
-    set_seed(SEED)
-    generation = model.generate(
-        input_ids=inputs.input_ids, prompt_input_ids=prompt.input_ids, do_sample=True, temperature=1.0
-    )
-    audio_arr = generation.cpu().numpy().squeeze()
+        set_seed(SEED)
+        generation = model.generate(
+            input_ids=inputs.input_ids, prompt_input_ids=prompt.input_ids, do_sample=True, temperature=1.0
+        )
+        audio_arr = generation.cpu().numpy().squeeze()
```

Comment:
Perhaps easier to just do 

```python
@torch.no_grad()
def gen_tts(text, description):
    inputs = tokenizer(description, return_tensors="pt").to(device)
    ...
```

Reply:
Good suggestion, I'll update the code accordingly.

Reply:
Good idea! I'll make that change.

Reply:
Good suggestion! I will make that change.

Reply:
Good suggestion! I'll update the code to use the `@torch.no_grad()` decorator for `gen_tts`.